### PR TITLE
Updated User Guide

### PR DESCRIPTION
**Pull request by tzihiang**
Updated user guide to better reflect our product,

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -31,95 +43,127 @@ Jump to <<Quick Start>> to get started. Enjoy!
 .  Copy the file to the folder you want to use as the home folder for your Cooking Papa.
 .  Double-click the file to start the app. The GUI should appear in a few seconds.
 +
+.CookingPapa User Interface
 image::Ui.png[width="790"]
+[caption="Figure 1: ]
```
Comment (gohkh) [inline docs/UserGuide.adoc]: I think we don't need to put caption for images associated to a step?

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -31,95 +43,127 @@ Jump to <<Quick Start>> to get started. Enjoy!
...
 
 ==== View a recipe from the cookbook
 This command allows you to view a recipe, the ingredients required, and the steps to cook it.
 
 - Format: `cookbook view INDEX`
 - Example:
-* `cookbook view 1` +
-    Views the details of recipe 1.
+|===
+| Parameters | Result
```
Comment (gohkh) [inline docs/UserGuide.adoc:95]: Maybe we can change "Parameters" to "Examples" instead?

**Review by gohkh — APPROVED** [review]

File: docs/UserGuide.adoc (hunk 1 above)
Comment (teo-jun-xiong) [reply docs/UserGuide.adoc]: I think if we label it as figure 1, we should explicitly label what it shows too. But I think if the picture is for a step, then there may not be a need for it

File: docs/UserGuide.adoc (hunk 2 above)
Comment (tzihiang) [reply docs/UserGuide.adoc:95]: The entire table is shown under example, so parameters are the examples of parameters. Hope that makes sense!

**Review by teo-jun-xiong — APPROVED** [review]

File: docs/UserGuide.adoc (hunk 2 above)
Comment (gohkh) [reply docs/UserGuide.adoc:95]: or would "Command" be better? cus parameters seems like it's only referring to the arguments

**Review by hans555 — APPROVED** [review]
looks good

**Pull request merged**